### PR TITLE
Handle session logouts gracefully.

### PR DESCRIPTION
**Pull request by jtk54**
Handle session logouts gracefully. @ewiseblatt PTAL. @ttomsu FYI.

### CODE REVIEW

File: citest/gcp_testing/google_session.py [hunk 1]
```diff
@@ -69,13 +77,17 @@ def __init__(
         approval_dict[input['name']] = input['value']
     approval_dict['submit_access'] = 'true'
     self.__session.post(approval_form['action'], data=approval_dict)
```
Comment (ewiseblatt) [inline citest/gcp_testing/google_session.py]: I'm not diggin' the logout_url here or the flow in logout(). At least not now. It is hooking things together without being clear with what this means or is for or why the caller doesnt. On its own, this is authenticating a session. It isnt intrinsically on behalf of another party. So to logout for another party is just weird, and adding a registry of people to notify is overkill without more justification.

Comment (ewiseblatt) [inline citest/gcp_testing/google_session.py]: discussed offline. ok.

File: citest/gcp_testing/google_session.py [hunk 2]
```diff
@@ -87,4 +99,6 @@ def cookies(self):
       session as a CookieJar object.
       CookieJar documentation: https://docs.python.org/2.7/library/cookielib.html#cookiejar-and-filecookiejar-objects
     """
+    if not self.__session.cookies:
```
Comment (ewiseblatt) [inline citest/gcp_testing/google_session.py]: is None? Might not matter since you'll always have non-empty cookies otherwise?

Comment (jtk54) [inline citest/gcp_testing/google_session.py]: Done, it's theoretically possible to have zero cookies after login so we'll explicitly check it this way.

**Pull request merged**